include/tvSettings.h

```diff
@@ -2567,6 +2567,58 @@ tvError_t EnableDynamicContrast(bool mode);
  */
 tvError_t EnableLocalContrast(bool mode);
 
+/** 
+* @brief Triggers short circuit detection operation and returns the status.
+*
+* On calling GetLdimZoneShortCircuitStatus, the short circuit detection logic
+* is triggered. It detects short circuits between zones.  On completion, it
+* returns a list of zones where a short circuit was detected.
+*
+* @param[in/out] status
```

Comment:
Add @param[out]. Dont represent as [in/out]

Reply:
Ok, will update it as out param